test/functional/fixtures/concurrency/testcafe-fixtures/concurrent-fixture-before-test.js

```diff
@@ -0,0 +1,31 @@
+import testInfo from '../test-info.js';
+
+let value = 0;
+
+fixture('Concurrent fixture before hook')
+    .before(async () => {
+        testInfo.add('fixture before hook started');
```

Comment:
As far as I understand you want to check that fixture before hook is called only once. You do not need to use `testInfo` for this purpose. It's enough to add some counter `let beforeHookCallNumber = 0` and increase it on the `before` hook. Then on `after` hook you can check that `beforeHookCallNumber` equals 1.